kernel/common/parse_ip.rs

```diff
@@ -23,13 +23,9 @@ pub fn parse_port(string: &str) -> &str {
 
 /// Get the host from a string (ip)
 pub fn parse_host(string: &str) -> &str {
-    // TODO: Username/Password syntax
-    let pos = match string.find(':') {
+    let pos = match string.find(|c| c == ':' || c == '/') {
```

Comment:
How cool, I didn't know you could use a closure inside `.find()` for multiple checks.

Thanks for backing me up. :smile: